src/Components/Search/SearchBar.tsx

```diff
@@ -175,38 +181,46 @@ export class SearchBar extends Component<Props, State> {
   }
 
   renderInputComponent = inputProps => (
-    <Box>
-      <Input style={{ width: "100%" }} {...inputProps} />
-    </Box>
+    <Input style={{ width: "100%" }} {...inputProps} />
   )
 
   renderAutosuggestComponent({ xs }) {
-    const { input } = this.state
+    const { term } = this.state
     const { viewer } = this.props
-    const edges = get(viewer, v => v.search.edges, [])
 
     const inputProps = {
       onChange: this.searchTextChanged,
       onFocus: this.onFocus,
       onBlur: this.onBlur,
       placeholder: xs ? "" : PLACEHOLDER,
-      value: input,
+      value: term,
     }
 
+    const firstSuggestionPlaceholder = {
```

Comment:
Here's the 'fake' first item, which really needs the `displayLabel` (ie- what to display), and the `href` (where to navigate when selected).

The `searchableType` is to distinguish it and be able to render it properly. But really it could be anything, I wasn't sure if it was more/less weird to use one that already exists in the payload.

Reply:
I really like how this turned out - good idea!! 🏆